bot/seasons/evergreen/error_handler.py

```diff
@@ -34,6 +39,16 @@ def revert_cooldown_counter(command, message):
 
         error = getattr(error, 'original', error)
 
+        if isinstance(error, InChannelCheckFailure):
+            logging.debug(
+                f"{ctx.author} the command '{ctx.command}', but they did not have "
+                f"permissions to run commands in the channel {ctx.channel}!"
+            )
+            embed = Embed(colour=Colour.red())
+            embed.title = random.choice(NEGATIVE_REPLIES)
+            embed.description = str(error)
+            return await ctx.send(embed=embed)

```

Comment:
Using an embed is not consistent with the rest of the errors which are just plain messages.

Reply:
I wanted it to be consistent with Python bot's wrong channel error message. Do you think the rest of the errors should be converted to an embed (they're not long error messages) to be consistent with Python bot? Or should we keep SeasonalBot's errors as a regular message?

Reply:
I'm not sure. This issue is still open python-discord/bot#126 and it's not entirely clear if we want to abolish embeds but in recent PRs it seems to have been going in that direction. Maybe best to leave this alone and come back to it later. At the least I can say you shouldn't change everything else to an embed.

Reply:
I have left this alone, the rest has been updated.